Modules/_datetimemodule.c

```diff
@@ -3234,6 +3254,9 @@ date_isocalendar(PyDateTime_Date *self, PyObject *Py_UNUSED(ignored))
     int  week;
     int  day;
 
+    PyObject* v = PyStructSequence_New(&StructIsoCalendarDateType);
+    if (!v) return NULL;
```

Comment:
The `return` should not be on the same line as the `if`.
```c
if (v == NULL) {
    return NULL;
}
```
Please see the [PEP 7](https://www.python.org/dev/peps/pep-0007/) style guide.